pkg/base1/cockpit.js

```diff
@@ -3905,7 +3905,7 @@ function factory() {
     var po_data = { };
     var po_plural;
 
-    cockpit.language = undefined;
+    cockpit.language = "en";
```

Comment:
It is being set on line 3928 (with default being picked on line 3911.
I am not sure if this is the right fix, as if you don't call `cockpit.locale()` (done in `pkg/lib/cockpit-po-plugin.js`) then you don't have any translation data really.

Reply:
Correct, I also mention it the commit message. But we don't really need to call cockpit.locale for english since there is not any translation really

Reply:
So now that we know what's going on, we could ship a stub po.js again, or just take your fix, which seems a bit more elegant to me. But then please drop the default three lines below, to not duplicate it.